3ds2/src/main/java/com/adyen/checkout/adyen3ds2/internal/ui/DefaultAdyen3DS2Delegate.kt

```diff
@@ -97,8 +97,11 @@ internal class DefaultAdyen3DS2Delegate(
 
     private var authorizationToken: String? by SavedStateHandleProperty(AUTHORIZATION_TOKEN_KEY)
 
+    private var action: BaseThreeds2Action? by SavedStateHandleProperty(ACTION_KEY)
```

Comment:
Is storing `action` necessary, since `action` is always passed in `handleAction()`?

Reply:
Yes, it seems like we don't do anything with it, but it's the opposite. By storing the action `DefaultGenericActionDelegate` will be able to recreate the 3ds delegate